packages/app/package.json

```diff
@@ -13,64 +13,71 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "cmd=\"ember build --environment=$npm_config_environment\" && echo $cmd && eval $cmd ",
+    "build": "ember build --environment=production",
```

Comment:
Did we update passing the environment in the `webservice` package?

Reply:
Nope, didn't catch this script update